cdap-common/src/main/java/io/cdap/cdap/common/http/AuthenticationChannelHandler.java

```diff
@@ -89,7 +90,7 @@ public void channelRead(ChannelHandlerContext ctx, Object msg) throws Exception
         if (idx < 0) {
           LOG.error("Invalid Authorization header format for {}@{}", currentUserID, currentUserIP);
           if (internalAuthEnabled) {
-            throw new IllegalArgumentException("Invalid Authorization header format");
+            throw new UnauthorizedException("Invalid Authorization header format");
```

Comment:
Can we use the existing [`UnauthenticatedException`](https://github.com/cdapio/cdap/blob/develop/cdap-security-spi/src/main/java/io/cdap/cdap/security/spi/authentication/UnauthenticatedException.java) here instead of creating a new exception?